### ISSUE TITLE
Add Upload button to Media Selection Overlay

### ISSUE THREAD

**Issue by alexander-schranz** [sulu/sulu#5039]
# Problem description

Not all end users are familiar with Drag&Drop so this users think they have to always change to the media module to upload new files to sulu instead of Drag&Drop them into the overlay.

I personally example does not use Drag&Drop when using Windows as you mostly have all Windows in Fullscreen mode which is on Mac something you mostly don't have. So think windows users are mostly not so familiar with Drag&Drop because its not so intergrated in there OS or other Software there.

# Proposed solution

The Media Selection Overlay should also show a upload button a user can click, so the users which are not familiar with Drag&Drop can use that.

**Comment by danrot** [comment]
Generally I am fine with that, I just don't know where we should place the upload button :thinking:

We could place it somewhere in the `MediaCollection` overlay, but I think then we would have to change that also in the media module.

The other option would be to integrate the `ToolbarActions` somehow in the Overlay as well. But we have no design for that, and I am not really sure how that would fit, because the heading of the overlay would probably have to go somewhere else...

**Comment by Warxcell** [comment]
That's my opinion.

![image](https://user-images.githubusercontent.com/3340882/74256216-b11a1400-4cfb-11ea-9746-6f47da2d1302.png)

**Comment by danrot** [comment]
That's a possibility, but IMO then we would also have to move the button in the media section of Sulu itself (you get there by clicking the media item in the navigation).

**Comment by Warxcell** [comment]
Yup I guess. I think even this dropdown could be the same way. There is a plenty of space. 

![image](https://user-images.githubusercontent.com/3340882/74331027-7798e580-4d9b-11ea-97d1-3b3c7e28ac68.png)

**Comment by jordygroote** [comment]
I also expierenced this. As of now it’s somewhat unclear that it is possible to drag and drop.

**Comment by jordygroote** [comment]
@alexander-schranz Any insight on when this is fixed?

**Comment by hedwiggggg** [comment]
Came across the same issue today, could create a PR.
Something like this?


https://user-images.githubusercontent.com/42947316/111484451-1083f800-8736-11eb-92c4-23320db5d3a9.mp4

**Comment by alexander-schranz** [comment]
We did get today a Design from our UX Team, with the changes for the Collection / Media view:

![Upload Image](https://user-images.githubusercontent.com/1698337/111486063-71f89680-8737-11eb-97f9-6e7907caf41b.png)

This button will also be visible in the normal collection view:

![All Media](https://user-images.githubusercontent.com/1698337/111486419-c3088a80-8737-11eb-9316-b59de7fd6240.png)

So the upload button will be at the bottom part of the view where the medias are listed not on the collection. 

There is also some small change for the `+` to replace with a `Add Folder +` Button at the to to also improve the UX there. That don't need to be done in the same PR. We also got a design for the Issue about uploading Media without a collection setting: #5038 but that is an own issue.

@hedwiggggg if you want to give it a try feel free to create a PR.
